site/content/en/docs/contribution guidelines/_index.md

```diff
@@ -6,7 +6,7 @@ description: >
   How to contribute to Jobset
 ---
 
-Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://git.k8s.io/community)! The Kubernetes community abides by the CNCF [code of conduct](code-of-conduct.md). Here is an excerpt:
+Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://git.k8s.io/community)! The Kubernetes community abides by the CNCF [code of conduct](../code-of-conduct.md). Here is an excerpt:
```

Comment:
The code of conduct link still doesn't work in the preview

Reply:
thank for catching that. fix!

Reply:
Let's just have a copy of the code-of-conduct.md in the site/ directory somewhere so we don't need to use this super long relative path

Reply:
I referred to the site/content/en/docs/reference directory and also placed the code-of-conduct.md in the site/content/en/docs/contribution guidelines directory

Reply:
https://github.com/googs1025/jobset/blob/jobset_sit/site/content/en/docs/contribution%20guidelines/_index.md
This is a bit strange. It works on my branch.
![Uploading image.png…]()

Reply:
Maybe the preview link here is outdated? https://github.com/kubernetes-sigs/jobset/pull/541#issuecomment-2067821199 Not sure, it is still live though so seems like it should reflect the latest changes